components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx

```diff
@@ -3,20 +3,11 @@ import { useGesture } from "@use-gesture/react";
 import Image from "next/image";
 import { ReactNode, useState } from "react";
 import * as RawTooltip from "@radix-ui/react-tooltip";
-import Button from "components/atoms/Button/button";
```

Comment:
This file is refactored in this PR because there is a shared component now, `ContributorTypeFilter`.